pkg/ha/filter.go

```diff
@@ -13,35 +13,39 @@ import (
 	"github.com/timescale/promscale/pkg/prompb"
 )
 
-const ReplicaNameLabel = "__replica__"
-const ClusterNameLabel = "cluster"
+const DefaultReplicaLabelName = "__replica__"
+const DefaultClusterLabelName = "cluster"
```

Comment:
I think the better place for these constants is pkg/ha/config.go.

Reply:
Agreed.